lib/puma/dsl.rb

```diff
@@ -241,6 +241,10 @@ def tcp_mode!
       @options[:mode] = :tcp
     end
 
+    def early_hints!
```

Comment:
I know our DSL style is really inconsistent, but for futureproofing I'd prefer:

```ruby
def early_hints(answer=true)
    @options[:early_hints] = answer
end
```

Reply:
Fixed 👍